Gemfile

```diff
@@ -12,7 +12,7 @@ group :development, :test do
   gem 'pry'
   gem 'rake'
   gem 'rspec'
-  gem 'rubocop', '~> 0.61.1'
+  gem 'rubocop'
```

Comment:
Put back the version of rubocop.

Reply:
Done.

But, note rubocop -a and rubocop --auto-gen-config did not fix my issues with "Style/FrozenStringLiteralComment" messages when running  'bundle exec rake' 

I was able to run specific gear spec after modification to it though. Seems 'bundle exec rake'  should not be advised? Also, if you force Rubcop version, it might not work depending on installed Ruby version?